src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -73,16 +75,20 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
         } catch (error) {
             const parsedError: IParsedError = parseError(error);
 
-            if (parsedError.errorType !== 'UserCancelledError') {
-                // tslint:disable-next-line: no-floating-promises
-                ext.ui.showWarningMessage(localize('couldNotConnect', 'Could not connect to "{0}": {1}', this.parent.label, parsedError.message));
+            if (parsedError.errorType === invalidCredentialsErrorType || parsedError.errorType === 'UserCancelledError' || parsedError.errorType === 'PromptForCredentialsError') {
```

Comment:
No need to check `UserCancelledError` anymore since you're not prompting for credentials